python/ribasim/ribasim/input_base.py

```diff
@@ -515,15 +514,7 @@ def _from_db(cls, path: Path, table: str):
         with closing(connect(path)) as connection:
             if exists(connection, table):
                 # pyogrio hardcodes fid name on reading
-                df = gpd.read_file(
-                    path,
-                    layer=table,
-                    engine="pyogrio",
-                    fid_as_index=True,
-                    use_arrow=True,
```

Comment:
Might be good to check/confirm that use_arrow/engine is now default? I remember using arrow was way faster, and actually using the arrow dtypes was a bit faster on top of that. Would be good to keep the initial performance gain.

Reply:
Yeah good point, pyogrio is default if installed, arrow is not: 6e9365ee164bf1f5d5cf404b283bd628ee52e2f8